components/bl-stepper-component/src/helpers/use-stepper-class-name.js

```diff
@@ -0,0 +1,40 @@
+const classes = {
+  root: {
+    stepper: 'bl-customComponent-stepper',
+    item: 'stepper__item',
+    stepTitle: 'stepper__step-title step-title',
+    line: 'stepper__line',
+    titleIcon: 'step-title__icon',
+    titleIconActive: 'step-title__icon--active',
+    titleIconText: 'step-title__icon-text',
+    titleText: 'step-title__text',
+    titleTextActive: 'step-title__text--active',
+  },
+  alternativeLabel: {
+    stepper: 'bl-customComponent-stepper bl-customComponent-stepper--stepper-alternative-label',
```

Comment:
`bl-customComponent-stepper--stepper-alternative-label`
`bl-customComponent-stepper--alternative-label`

It's different stepper types, but classes looks pretty similar.